### PR TITLE
Replace `sv_pipeline_rdtest_docker` with `sv_pipeline_docker`.

### PR DESCRIPTION
**Pull request by VJalili**
A follow-up from #633, similar to #634 and #639.

This PR improves input definition by collapsing identical inputs defined by different variables and decreasing the number of inputs the workflows take. Specifically, it implements the following changes: 

- Replace all the references to `sv_pipeline_rdtest_docker` with `sv_pipeline_docker`; and,
- Remove `sv_pipeline_rdtest_docker` if `sv_pipeline_docker` is also among the inputs to the workflow.

### CODE REVIEW

**Comment by MattWellie** [comment]
very happy this is being addressed

**Review by mwalker174 — APPROVED** [review]

File: inputs/templates/test/Module09/Module09VisualizeSingleSample.json.tmpl [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
         "Module09VisualizeSingleSample.sv_base_mini_docker" : "us.gcr.io/broad-dsde-methods/gatk-sv/sv-base-mini:mw-gnomad-02-6a66c96",
-        "Module09VisualizeSingleSample.sv_pipeline_rdtest_docker" : "us.gcr.io/broad-dsde-methods/gatk-sv/sv-pipeline-rdtest:mw-gnomad-02-6a66c96",
+        "Module09VisualizeSingleSample.sv_pipeline_docker" : "us.gcr.io/broad-dsde-methods/gatk-sv/sv-pipeline:2024-04-29-v0.28.4-beta-7fd43657",
```
Comment (VJalili) [inline inputs/templates/test/Module09/Module09VisualizeSingleSample.json.tmpl:3]: ping @mwalker174 Would you still prefer to use the `sv-pipeline-rdtest` image here or rather use the current latest build?

Comment (mwalker174) [reply inputs/templates/test/Module09/Module09VisualizeSingleSample.json.tmpl:3]: Use the latest

Comment (VJalili) [reply inputs/templates/test/Module09/Module09VisualizeSingleSample.json.tmpl:3]: Sounds good, thanks!

**Pull request merged**